pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -57,13 +59,19 @@ const (
 
 	// LocalhostAddress for local binding
 	LocalhostAddress = "127.0.0.1"
+
+	traceSamplingDefault = 100.0
```

Comment:
Same - I wouldn't default to 100%, either 0 or a small percent.

Reply:
People are used to 100% so far (migration). Not having 100% is going to be a big surprise. We can specify sensible defaults for a variety of things in Operations Guide. Any other value is purely arbitrary (10% vs 1% vs 50%). And for systems with low traffic (10-20 Queries per hour), 100% is the way to go. While those with high traffic will want to bring this down lower. Since we don't know who is going to install, any value <100 will always be arbitrary.